src/python/turicreate/test/test_audio_functionality.py

```diff
@@ -28,7 +28,7 @@
 from turicreate.toolkits._internal_utils import _mac_ver
 
 
-IS_PRE_6_0_RC = float(tc.__version__) < 6.0
+IS_PRE_6_0_RC = float(tc.__version__[:3]) < 6.0
```

Comment:
Should we use [packaging.version.parse](https://packaging.pypa.io/en/latest/version/#packaging.version.parse)?

Reply:
Actually, now that this is 6.0rc0, shouldn't we just do `IS_PRE_6_0_RC = false` (and file a bug to clean it up later), or just remove the pre-6.0 code now?